core-plugins/widgets/Cube-realtimesink.json

```diff
@@ -35,6 +46,17 @@
       "label": "CubeFact Mapping",
       "properties": [
         {
+          "widget-type": "keyvalue-dropdown",
+          "label": "Measurements",
+          "name": "cubeFact.measurements",
+          "description": "Fields that are used as measurements and the measurment type.",
```

Comment:
Do we need more info about this? For example, say I add couple of aggregations each containing one field each. Now should I also provide measurements for each of those fields? What if I don't? What does it mean if I have measurements for some field and it is not present in aggregation?